harvard-lts/CURIOSity
# Removing <h1> title from show pages with associated css.

**Removing h1 title tag from show pages with associated css.**
* * *

**JIRA Ticket**: [LTSMAINT-768](https://jira.huit.harvard.edu/browse/LTSMAINT-768)

# Other Relevant Links (Mailing list discussion, related pull requests, etc.)
* Slack thread in code4lib #blacklight channel about this solution: https://code4lib.slack.com/archives/C54TB5WDQ/p1687883505643259

# What does this Pull Request do?
This removes the h1 tag above the Mirador viewer on document show pages that displays the title. It was previously hidden with css, but was still showing up in screen readers. 

# How should this be tested?

A description of what steps someone could take to:
* Rebuild CURIOSity using the `LTSMAINT-768` branch
* Go to any item in any exhibit. For example: https://localhost:21407/contagion/catalog/36-990006033660203941_HMSCOUNT:1204541
* Inspect the code in your browser console above the Mirador viewer. Confirm that there is no `<h1>` tag before the `<div class="iframe-wrapper">` tag.
* The above example on CURIOSity production (https://curiosity.lib.harvard.edu/contagion/catalog/36-990006033660203941_HMSCOUNT:1204541) has the tag, which looks like this: `<h1 itemprop="name">Lyman Spalding papers, 1798-1912 (inclusive), 1798-circa 1820 (bulk), Draft of a letter to Benjamin Waterhouse, 1800. B MS c2.1</h1>`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @mferrarini
